com.unity.render-pipelines.high-definition/Editor/ToolMenus.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEditor.Rendering.Utilities;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    class ToolMenus
+    {
+        [MenuItemForRenderPipeline("Window/Rendering/Look Dev", false, 10001, typeof(HDRenderPipelineAsset))]
+        static void OpenLookDev()
+            => LookDev.LookDev.Open();
+
+        [MenuItemForRenderPipeline("Window/Analysis/Render Graph Viewer", false, 10006, typeof(HDRenderPipelineAsset))]
```

Comment:
Priority should be the same,  right? 10001 vs 10006

Reply:
Priority for RenderGraph (Viewer) was 1006. I keep what was used.